README.md

```diff
@@ -107,6 +107,9 @@ Options:
   -h, --help                       display help for command
 ```
 
+### Command line npm package (CLI)
+The gbfs-validator CLI Node.js npm package is now accessible for integration into your projects. To learn how to install and utilize it effectively, please refer to the [README](./gbfs-validator/README.md) for comprehensive guidance.
```

Comment:
The npm package not only contains the Command Line Interface (CLI) but also contains the core GBFS validator libraries. These libraries can be used directly from JavaScript and also as a CLI command.
```suggestion
### Npm package 
The gbfs-validator Node.js npm package is now accessible for integration into your projects. To learn how to install and utilize it effectively, please refer to the [README](./gbfs-validator/README.md) for comprehensive guidance.
```